doc/api/cli.md

```diff
@@ -490,6 +490,40 @@ For example, to run a module with "development" resolutions:
 node -C development app.js
```

Comment:
There is a "Limitations" section in the `node:test` code coverage documentation. This was the last listed limitation, so that section should be removed.